Assets/Scripts/GameController.cs

```diff
@@ -368,6 +409,26 @@ public class GameController : MonoBehaviour {
 				Debug.Log("Endturn canceled");
 			}
 		}
+		if(!GameData.CurrentMatch.UserTurn) {
+			_qguTimer += Time.deltaTime;
+			if(_qguTimer >= _qguInterval) {
+				CheckForTurn();
+				_qguTimer = 0f;
+			}
+		}
+	}
+
+	// Loads active games
+	private void CheckForTurn() {
+		if(!(bool)Server.QueryGames()["Success"]) {
+			Debug.Log("Query Games failed");
```

Comment:
We can, though the next line of code checks for UserTurn which will still be false if QGU failed and so nothing will happen anyways